TAO/orbsvcs/ImplRepo_Service/AsyncAccessManager.cpp

```diff
@@ -283,11 +283,27 @@ AsyncAccessManager::update_status (ImplementationRepository::AAM_Status s)
 }
 
 void
-AsyncAccessManager::activator_replied (bool success)
+AsyncAccessManager::activator_replied (bool success, pid_t pid)
 {
   if (success)
     {
-      this->update_status (ImplementationRepository::AAM_WAIT_FOR_RUNNING);
+      if (pid == 0)
```

Comment:
No, pid use is optional inside the locator based on context. Early on in this generation of the ImR pids were not used at all. To distinguish between "pid expected but not valid" and "pid value not available" I use 0 for the latter case.